app/src/main/java/net/skyscanner/backpack/demo/MainActivity.kt

```diff
@@ -41,21 +64,90 @@ class MainActivity : BpkBaseActivity() {
 
   override fun onCreate(savedInstanceState: Bundle?) {
     super.onCreate(savedInstanceState)
-    setContentView(R.layout.activity_component_list)
+    setContent {
+      BpkTheme {
+        Column {
+          val context = LocalContext.current
+          BpkTopNavBar(
+            navIcon = NavIcon.None,
+            title = stringResource(R.string.app_name),
+            actions = listOf(
+              IconAction(icon = BpkIcon.Settings, contentDescription = stringResource(R.string.settings_title), onClick = {
```

Comment:
Can we move the arguments to separate lines?

Reply:
Thanks, fixed this!